src/Routes/Devices/DeviceTable.js

```diff
@@ -159,9 +160,17 @@ const createRows = (devices, hasLinks) => {
           title: LastSeen ? <DateFormat date={LastSeen} /> : 'Unknown',
         },
         {
-          title: (
-            <DeviceStatus type={getDeviceStatus(Status, UpdateAvailable)} />
-          ),
+          title:
+            getDeviceStatus(Status, UpdateAvailable) !== 'updateAvailable' ? (
+              <DeviceStatus type={getDeviceStatus(Status, UpdateAvailable)} />
```

Comment:
Please add a comment stating that this co temporary until FE implements the Error status.

# FIXME: change `UpdateAvailable` to `Error` status when it is available on the Frontend. Related to THEEDGE-1449